client/tailscale/localclient.go

```diff
@@ -1028,6 +1028,22 @@ func (lc *LocalClient) DebugSetExpireIn(ctx context.Context, d time.Duration) er
 	return err
 }
 
+// StreamDebugCapture streams a pcap-formatted packet capture.
+func (lc *LocalClient) StreamDebugCapture(ctx context.Context) (io.Reader, error) {
+	req, err := http.NewRequestWithContext(ctx, "POST", "http://"+apitype.LocalAPIHost+"/localapi/v0/debug-capture", nil)
+	if err != nil {
+		return nil, err
+	}
+	res, err := lc.doLocalRequestNiceError(req)
+	if err != nil {
+		return nil, err
+	}
+	if res.StatusCode != 200 {
+		return nil, errors.New(res.Status)
+	}
+	return res.Body, nil
```

Comment:
this is a ReadCloser so you should probably return one and document that the ctx is only used for the initial return and not the lifetime of the reader content.